### PR TITLE
Project: Setup boilerplate for Project (CSS/JS)

### PR DESCRIPTION
**Pull request by PatricNox**
Setting up boilerplate for project structure and development workflow.

This embraces SCSS (src/styles) and JS (src/scripts) for a cleaner work space and easier workflow.

### CODE REVIEW

**Comment by PatricNox** [comment]
Even here we could use collaborations to test this out and update readme to become more of a newbie guideline to how to work with this 😄

File: src/scripts/main.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/**  Main JS file
+*
+* Following file is the base file for all the functionality scripting..
+* Components goes under corresponding map, binds together through here.
+*
+*/
+
+console.log('Javascript is compilled!');
```
Comment (BenjaminGarza) [inline src/scripts/main.js]: Typo on line 8, "compiled"

Comment (BenjaminGarza) [inline src/scripts/main.js]: Small typo on line 8, "compiled"

**Comment by kirubiel** [comment]
Haha on the next PR then

File: src/scripts/main.js (hunk 1 above)
Comment (PatricNox) [reply src/scripts/main.js]: Woops!

This line was to make it easier for devs to see that JS actually got compiled properly anyways 😁

**Pull request merged**